chainer/variable.py

```diff
@@ -867,6 +867,24 @@ def array(self, d):
             self._data[0] = d
             self._has_chainerx_array = False
 
+    def chx_view(self):
```

Comment:
Could you clarify why it's returning a view?

Reply:
If it is not, `var.chx_view().require_grad()` would affect the original array, making `var` in an invalid state.

Reply:
Thanks for the explanation.